Firestore/Example/Tests/Util/FSTDispatchQueueTests.mm

```diff
@@ -66,9 +66,8 @@ - (void)testDispatchAsyncBlocksSubmissionFromTasksOnTheQueue {
   XCTAssertNotNil(caught);
 
   XCTAssertEqualObjects(caught.name, NSInternalInconsistencyException);
-  XCTAssertTrue(
-      [caught.reason hasPrefix:@"FIRESTORE INTERNAL ASSERTION FAILED: "
-                               @"dispatchAsync called when we are already running on target"]);
+  XCTAssertTrue([caught.reason hasPrefix:@"FIRESTORE INTERNAL ASSERTION FAILED: "
+                                         @"Enforcing sequential order failed"]);
```

Comment:
The full error message is _"Enforcing sequential order failed: currently executing operations cannot enqueue more operations "_. I just omitted a part of the assertion because I thought it's not necessary to be so specific for the test. Part of the reason I omitted `dispatchAsync` bit is because the same check is applied to `dispatchSync` -- OTOH, the latter is a second-class citizen, so perhaps it's not as important. If you think the full message is still less clear, I'll rework it.

Reply:
I understand your motivation. Maybe something like "Enqueue methods" -- a plural that suggests more than just exactly "Enqueue" -- would be a suitable subject for a sentence more similar to the old one?

This would at least more directly suggest a binding between the action that was taken and the failure. As it is you need to understand that the AsyncQueue enforces some property and then find how your code may have interacted with that property to ultimately determine which call was problematic.

Reply:
Rephrased to be very similar to the previous version - " _Enqueue methods cannot be called_ when we are already running on target _executor_".